tests/run/enum-Color.scala

```diff
@@ -0,0 +1,13 @@
+enum Color {
+  case Red
+  case Green
+  case Blue
+}
```

Comment:
`enum Color { case Red, Green, Blue }`  is the word "case" needed in the abbreviation?